packages/preset-wind/src/rules/background.ts

```diff
@@ -22,6 +22,11 @@ const bgGradientColorValue = (mode: string, cssColor: CSSColorValue | undefined,
 
 const bgGradientColorResolver = (mode: 'from' | 'to' | 'via') =>
   ([, body]: string[], { theme }: RuleContext<Theme>) => {
+    if (/^[0-9]/.test(body)) {
+      return {
+        [`--un-gradient-${mode}-position`]: h.bracket(`[${body}]`)
```

Comment:
You may want to do var reset since using this without stops now may generate unset variable.

Reply:
I have a problem the default value will cover the set value. How to resolve it?

<img width="328" alt="image" src="https://user-images.githubusercontent.com/57086651/229357049-4c2234bc-19d8-454f-8db1-b1915db28895.png">

Reply:
I fix it through add another rule